validator/cpp/htmlparser/BUILD

```diff
@@ -555,3 +555,14 @@ cc_test(
         "@com_google_absl//absl/flags:flag",
     ],
 )
+
+cc_test(
```

Comment:
Consider adding the `FUZZ_TEST` in the existing `validator/cpp/htmlparser/parser_test.cc` test source along with the regular `TEST`-s.

Reply:
Good idea -- Am happy to do so!

AMP maintainers: do you have a preference in this context?

Reply:
cc @amaltas , could we get your opinion on this.